harmony-signing/sign_binary.py

```diff
@@ -0,0 +1,53 @@
+import aws_encryption_sdk
+import boto3
+import sys
+import hashlib
+import binascii
+
+#key ID for harmony signing key, created by aws, using aws KMS 
+harmonySigningKeyId = "arn:aws:kms:us-west-2:656503231766:key/370d45bb-d629-45d7-8d1c-db0560895690"
+signingAlgorithm    = "RSASSA_PKCS1_V1_5_SHA_256"
+
+def region_from_key_id(signingKeyId, default_region=None):
+    try:
+        region_name = signingKeyId.split(":", 4)[3]
+    except IndexError:
+        if default_region is None:
+            raise UnknownRegionError(
+                "No default region found and no region determinable from key id: {}".format(signingKeyId)
+            )
+        region_name = default_region
+    return region_name
+
+
+def sign_harmony_file(inputFile, sigFile, keyId = harmonySigningKeyId):
+    region_name = region_from_key_id(keyId)
+    kms_client = boto3.client('kms', region_name)
+
+    data = open(inputFile, "rb").read()
```

Comment:
not a big deal because the OS will let recycle the file descriptor for you, but usually best in python to use the with context when opening files, that is, 
```python
with open(inputFile, 'rb') as f:
    data = f.read()
```